polyfill/lib/timezone.mjs

```diff
@@ -80,7 +82,7 @@ export class TimeZone {
 
     if (!~['earlier', 'later'].indexOf(disambiguation)) throw new RangeError(`no such absolute found`);
 
-    const { ms: utcms } = ES.GetEpochFromParts(
+    const utcns = ES.GetEpochFromParts(
```

Comment:
Would this work?

Reply:
Yes, the existing code was broken but it wasn't covered by any tests. My guess is that the existing code was correct at one point but then the return value of `GetEpochFromParts` must have changed.